data/sssd-tests/testincl.sh

```diff
@@ -94,11 +96,16 @@ test_abort() {
 }
 
 
-isSles15(){
-   if [ ! -f /etc/os-release ]; then
-      return 1
-   fi
-   if (grep -i -q "SUSE Linux Enterprise Server 15" /etc/os-release); then
+#########################################################################
+###
+### Returns 0 if python3 should be used
```

Comment:
Can you please state more clearly why we want to know if python3 should be used?. Since you need to remember that there's quite a matrix on which systems do we support.

Reply:
I didn't make that decision. The test was already using Python3 for SLE15+.

Reply:
See https://pythonclock.org/ python2 is phased out. Newer distros are meant to use python3. I'm not sure I would detect it like that though. Better use a shebang line for python3 in the script. Only for legacy cases switch to python2 (like simply check if /usr/bin/python3 exists)

Reply:
not sure what his resolve button does. I'll leave that to you

Reply:
The problem with checking for python3 is that it can hide bugs.
As acceptance test, we should check that if we are on a system where we expect python3, we use it.

Reply:
@lnussel @SergioAtSUSE my comment was more about an actual comment in the code :), but we can go ahead and merge :P 

> not sure what his resolve button does. I'll leave that to you
Just makes the conversation dissapear, or allows us to pretend that the world is clear and without terrors